api/schemas/configs/kmeans_configs.py

```diff
@@ -0,0 +1,38 @@
+from dataclasses import dataclass
+from typing import Optional, Literal
+from configs.algorithms_configs import UnsupervisedAlgorithmsParams
```

Comment:
This should read from schemas.configs.algorithms_configs import UnsupervisedAlgorithmsParams